aion/web/elm/src/Ranking/View.elm

```diff
@@ -0,0 +1,50 @@
+module Ranking.View exposing (..)
+
+import General.Models exposing (Model)
+import Html exposing (..)
+import Msgs exposing (Msg(..))
+import Material.Table as Table
+import Bootstrap.Grid as Grid
+import Bootstrap.Grid.Col as Col
+import Ranking.Models exposing (RankingData)
+import RemoteData exposing (WebData)
+
+
+rankingView : Model -> Html Msg
+rankingView model =
+      Grid.container []
+          [ Grid.row []
+              [ Grid.col [ Col.xs4 ]
+                  [ h4 []  [ text "General" ] ] ]
+          , Grid.row []
+              [ Grid.col [ Col.xs4 ]
+                  [ rankingTable model ]
+              ]
+          ]
+
+
+rankingTable : Model -> Html Msg
+rankingTable model =
+    Table.table []
+        [ Table.thead []
+            [ Table.tr []
+                [ Table.th [] [ text "User" ]
+                , Table.th [] [ text "Score" ]
+              ]
+            ]
+        , Table.tbody [] (scores model.rankingData)
```

Comment:
I was confused for a longer while as to what the scores function could be.  The same thing as in elixir. How about `display_scores`? When I see `scores` my first thought is that it must be a variable containing some data structure with scores. Surprisingly it's a function rendering `Html Msg`

Reply:
Also, instead of returning `List(Html Msg) I think the conventional way to do stuff is to return `Html Msg`. If you're doing it this way and at any time in the future you want to reuse this part, you will impose the way the invoking function has to be exactly implemented. I guess you should at least move tbody to the scores function